extensions/arc/src/suggestions.ts

```diff
@@ -61,6 +61,11 @@ const config: SearchConfigs = {
       });
     },
   },
+  kagi: {
+    search: "https://kagi.com/search?q=",
+    suggestions: null, //Note: Unknown
```

Comment:
Can this use `https://kagi.com/api/autosuggest?q=%s` as mentioned on https://kagi.com/assets?

Reply:
Looks like it can! don't have the time to look into it rn but will try in a few days. 
For now I opened #10461 so we can track it.

ty so much